src/index.d.ts

```diff
@@ -115,6 +139,7 @@ declare module 'bootstrap-vue' {
   export interface Progress extends Vue {}
   export interface Table extends Vue {
     refresh: () => void
+    fields?: TableFieldObject | TableFieldArray
```

Comment:
So why is `fields` mentioned here, but `items` is not? should we define all the props, or is that just overkill?

Reply:
I just added `fields` because it's what I already had right now, and I thought it should go upstream.

Reply:
OK...  at some point in the future we should define the other props.